paf-mvp-cmp/README.md

```diff
@@ -1,108 +1,305 @@
 ```
 
-The DOM is modified at the point the script is included. There are no other deployment options.
-
-### Performance (2)
+If this link is not added, then the functionality will still be provided by page
+load time will be marginally impaired.
 
-By placing the script element for the CMP as close to the top of the `<body>` element as possible the interface will 
-appear on first request while the rest of the page is loading behind it. Should the interface require a redirect to 
-fetch or store data then the assets for the publishers page will already be cached by the web browser effectively 
-improving user perceive page load time. Also the approach is likely to improve actual page load time compared to 
-solutions which load the interface after other page processing completes.
+This technique can be further improved by modifying the `link` header of the
+HTTP response to specify the
+[preload](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/preload)
+hint.
 
-## Configuration options
+### Performance (3)
 
-The following attributes must be present in the script tag.
+By placing the script element for the CMP as close to the top of the `<body>`
+element as possible the interface will appear on first request while the rest of
+the page is loading behind it. Should the interface require a redirect to fetch
+or store data then the assets for the publisher’s page will already be cached by
+the web browser effectively improving user perceive page load time. Further, the
+approach is likely to improve actual page load time compared to solutions which
+load the interface after other page processing completes.
 
-- data-brand-name: The brand name to use throughout the user interface.
-- data-brand-privacy-url: This URL is needed to inform the user about the privacy policy of the brand.
-- data-display-intro: True to display the introduction card, or false to skip straight to the settings card after a possible redirect.
-- data-proxy-host-name: The host name to use when reading and writing data from the global storage.
-- data-site-only-cookie-tcf-core: The name of the cookie used to store the TCF core string. If not provided then the this site only option is not available.
-- data-snackbar-timeout-ms: The number of milliseconds to wait for the snackbar to disappear.
-- data-template-tcf-core-string: The [template TCF core string](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md?msclkid=5236f9f5c47b11ec8a04e36f3dd976c9#the-core-string) that will be used when generating the resulting TCF core string from the CMP. This project will change the Purposes Consents, Created, and LastUpdated fields of the provided value when writing the TCF core string to the cookie. See [IAB TCF](https://iabtcf.com/#/encode) web site to generate this value.
+# For Developers
```

Comment:
The intention here is to reference the IAB TL standard. They're not a contributor and it would be a misrepresentation to claim they were. The tools listed are commonly used to generate and test TCF strings. Could you resolve unless I've missed something?

Reply:
I think I didn't make myself clear: I was talking about the "**For developers**" section that I think, could be extracted to a dedicated file.